.github/workflows/ci.yaml

```diff
@@ -123,6 +123,14 @@ jobs:
         printf '```\n\n' >> "${GITHUB_STEP_SUMMARY}"
         printf '</details>\n' >> "${GITHUB_STEP_SUMMARY}"
 
+        ./scripts/coverage.sh markup --destdir=coverage-report
+
+    - name: Create coverage report artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: coverage-report-${{ matrix.ghc }}-${{ matrix.cache-key-prefix }}
```

Comment:
A very minor thing, but for jobs without a `cache-key-prefix`, this results in artifact names like `coverage-report-9.10.2-` with a trailing hyphen. I don't suppose there is an easy way to remove the trailing hyphen?